hack/config

```diff
@@ -18,6 +18,7 @@ KUBE_RBAC_PROXY_VERSION="4.15"
 
 KUBEVIRT_CONSOLE_PLUGIN_VERSION="v4.15.0"
 KUBEVIRT_CONSOLE_PROXY_VERSION="v0.0.9"
+WASP_VERSION="latest"
```

Comment:
please use a semver tag for our bump automation.

Reply:
I don't sure we want it under bump automation. On the other hand, we don't want floating tag either, to avoid pulling a missing digest.